script.js

```diff
@@ -18,21 +25,59 @@ if (data && data.length > 0) {
 }
 
 function AddTodo(todoText) {
-    const element = document.createElement("LI")
-    const elementText = document.createTextNode(todoText)
-    element.appendChild(elementText)
+        const element = document.createElement("LI");
+	const elementBtn = document.createElement("button");
+	const elementText = document.createTextNode(todoText);
+	var elementId = nameId+loop;
+	loop++;
+	elementBtn.id = elementId;
+	element.id = elementId;
+	elementBtn.type = "submit";
+	elementBtn.innerHTML = "x";
+	elementBtn.addEventListener("click",function(){ removeHandler(elementBtn.id); });
 
+	element.appendChild(elementText);
+	element.appendChild(elementBtn);
+
+	
     // Update the View
-    outputListElement.appendChild(element)
+	outputListElement.appendChild(element);
 
     // Update the Model
-    todoCollection.push(todoText);
-    localStorage.setItem(localStorageKey, JSON.stringify(todoCollection))
+        todoCollection.push(todoText);
+	localStorage.setItem(localStorageKey, JSON.stringify(todoCollection));
+	
+}
+function RemTodo(todoText) {
```

Comment:
Functions in Javascript are written in camelCase so please change
```
function RemTodo(todoText) {
```
into
```
function remTodo(todoText) {
```
The Same goes for other functions in the code